lib/rubocop/cop/cop.rb

```diff
@@ -212,10 +217,16 @@ def reference_url
         (url.nil? || url.empty?) ? nil : url
       end
 
+      def details
+        details = cop_config && cop_config['Details']
```

Comment:
`cop_config` can be `nil` in cases where there is no configuration at all for a cop -- I have an open PR which addresses this so that guard clauses are not necessary.